sprint-mission-7/schema.sql

```diff
@@ -0,0 +1,66 @@
+CREATE TABLE users (
+  id SERIAL PRIMARY KEY,
+  email VARCHAR(255) UNIQUE NOT NULL,
+  password TEXT NOT NULL,
+  nickname VARCHAR(20),
+  created_at TIMESTAMP DEFAULT NOW() NOT NULL,
+  updated_at TIMESTAMP DEFAULT NOW() NOT NULL
+);
+
+CREATE TABLE products (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(30) NOT NULL,
+  description TEXT NOT NULL,
+  price INTEGER NOT NULL,
+  onwer_id INTEGER REFERENCES users(id) ON DELETE CASCADE,
+  url TEXT NOT NULL,
+  created_at TIMESTAMP DEFAULT NOW() NOT NULL,
+  updated_at TIMESTAMP DEFAULT NOW() NOT NULL
+)
+
+CREATE TABLE articles (
+  id SERIAL PRIMARY KEY,
+  title VARCHAR(50) NOT NULL,
+  content TEXT NOT NULL,
+  url TEXT NOT NULL,
+  writer_id INTEGER REFERENCES usrs(id) ON DELETE CASCADE,
```

Comment:
```suggestion
  writer_id INTEGER REFERENCES users(id) ON DELETE CASCADE,
```
`articles` 테이블에서 외래키 참조 시 테이블명에 오타가 있습니다.